tests/integration-tests/test_runner.py

```diff
@@ -688,6 +702,17 @@ def _run_parallel(args):
         job.join()
 
 
+def get_config_arguments(args):
```

Comment:
the internal convention is to use the `_` prefix for private functions, that are only used in this python module.

Reply:
Will make changes.